src/components/portrait/portrait.spec.js

```diff
@@ -39,7 +39,7 @@ function renderDLS(element) {
 function renderClassic(element) {
   return TestRenderer.create(
     <ThemeProvider theme={ classicTheme }>
-      {element}
+      {React.cloneElement(element, { theme: classicTheme })}
```

Comment:
why do we need `React.cloneElement` here?

Reply:
no more clone